packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Protocol-level requirements
+---
+
+### Code flow
+
+For OIDC applications destined for the OIN, you are building a "web" application with a dedicated server-side back end that is capable of securely storing a Client Secret and sharing it through trusted back-channel connections. The Authorization Code Flow is the recommended flow for controlling access to an OIDC application in the OIN context.
+
+The app should be server-side because the request that exchanges the authorization code for a token requires a Client Secret, which has to be stored in your client. The server-side app also requires an End User because it relies on interaction with the End User's web browser to redirect the user and then receive the authorization code. The End User and the user agent do not see the access token or any part of the authentication response.
```

Comment:
```suggestion
The app should be server-side because the request that exchanges the authorization code for a token requires a Client Secret, which has to be stored by your client. The server-side app also requires an End User because it relies on interaction with the End User's web browser to redirect the user and then receive the authorization code. The End User and the user agent do not see the access token or any part of the authentication response.
```